docs/src/main/asciidoc/virtual-threads.adoc

```diff
@@ -550,6 +550,45 @@ public class LoomUnitExampleTest {
 }
 ----
 
+== Virtual thread metrics
+
+You can enable the Micrometer Virtual Thread _binder_ by adding the following artifact to your application:
+
+[source,xml]
+----
+<dependency>
+    <groupId>io.micrometer</groupId>
+    <artifactId>micrometer-java21</artifactId>
+</dependency>
+----
+
+This binder keeps track of the number of pinning events and the number of virtual threads failed to be started or un-parked.
+See the https://docs.micrometer.io/micrometer/reference/reference/jvm.html#_java_21_metrics[MicroMeter documentation] for more information.
+
+You can explicitly disable the binder by setting the following property in your `application.properties`:
+
+[source,properties]
+----
+# The binder is automatically enabled if the micrometer-java21 dependency is present
+quarkus.micrometer.binder.virtual-threads.enabled=false
+----
+
+In addition, if the application is running on a JVM that does not support virtual threads (prior to Java 21), the binder is automatically disabled.
+
+You can associate tags to the collected metrics by setting the following properties in your `application.properties`:
+
+[source,properties]
+----
+quarkus.micrometer.binder.virtual-threads.tags=tag_1=value_1, tag_2=value_2
+----
+
+To compile the application to native and include the Micrometer Virtual Thread binder, you need to add the following property in your `application.properties`:
+
+[source,properties]
+----
+quarkus.native.monitoring=jfr
```

Comment:
Actually, I added a new Build Item for this, sorry, I should have advertised it more. 
See: https://github.com/quarkusio/quarkus/blob/a36e483cb4d76bee68118b757fa4eb055c8d0d1f/extensions/opentelemetry/deployment/src/main/java/io/quarkus/opentelemetry/deployment/metric/MetricProcessor.java#L40

No need to document, just add this silently to the processor.

Reply:
Ah nice!